studio/hooks/queries/useOrganizationDetail.ts

```diff
@@ -4,13 +4,36 @@ import { Member } from 'types'
 import { API_URL } from 'lib/constants'
 
 export function useOrganizationDetail(slug: string) {
+  // Get org members
   const url = `${API_URL}/props/org/${slug}`
   const { data, error } = useSWR<any>(url, get)
-  const { members, products } = data ?? []
-  const anyError = data?.error || error
+  let { members, products } = data ?? []
+
+  // Get pending invite users
+  const pendingInviteUrl = `${API_URL}/organizations/${slug}/members/invite`
+  const { data: inviteData, error: inviteError } = useSWR<any>(pendingInviteUrl, get)
+
+  if (inviteData && inviteData.length > 0) {
+    // remap invite data to look like existing members data
+    const reMappedInvitedata = inviteData.map((x: any) => ({
+      is_owner: false,
+      invited_at: x.invited_at,
```

Comment:
if we want to add these 2 properties we need to update the `Member` interface in the `types` file as well
I'm adding 2 optional properties `invited_at` and `invited_id` for now